Workaround for make 4.0 compatibility
--- gnu-efi-3.0/gnuefi/Makefile.orig	2014-09-04 19:12:43.901772181 +0200
+++ gnu-efi-3.0/gnuefi/Makefile	2014-09-04 19:12:47.435105396 +0200
@@ -72,7 +72,7 @@
 clean:
 	rm -f $(TARGETS) *~ *.o $(OBJS)
 
-$(LIBDIRINSTALL):
+$(LIBDIRINSTALL)/:
 	mkdir -p $@
 
 .SECONDEXPANSION:
--- gnu-efi-3.0/inc/Makefile.orig	2014-09-04 22:01:21.638014047 +0200
+++ gnu-efi-3.0/inc/Makefile	2014-09-04 22:01:37.774679986 +0200
@@ -25,10 +25,10 @@
 
 	rm -f *~
 
-$(INCDIRINSTALL) \
-$(INCDIRINSTALL)/protocol \
-$(INCDIRINSTALL)/$(ARCH) \
-$(INCDIRINSTALL)/protocol/$(ARCH):
+$(INCDIRINSTALL)/ \
+$(INCDIRINSTALL)/protocol/ \
+$(INCDIRINSTALL)/$(ARCH)/ \
+$(INCDIRINSTALL)/protocol/$(ARCH)/:
 	mkdir -p $@
 
 .SECONDEXPANSION:
