--- gnu-efi-3.0.17/lib/Makefile.orig	2024-02-10 20:12:40.581584850 +0100
+++ gnu-efi-3.0.17/lib/Makefile	2024-02-10 20:15:18.440729653 +0100
@@ -68,13 +68,12 @@ SUBDIRS = ia32 x86_64 ia64 aarch64 arm m
 
 LIBDIRINSTALL = $(INSTALLROOT)$(LIBDIR)
 
-all: libsubdirs libefi.a
+all: .libsubdirs libefi.a
 
-.PHONY: libsubdirs
-libsubdirs:
-	@set -e ; for sdir in $(SUBDIRS); do mkdir -p $$sdir; done
+.libsubdirs:
+	@set -e ; for sdir in $(SUBDIRS); do mkdir -p $$sdir; done && touch .libsubdirs
 
-$(OBJS): libsubdirs
+$(OBJS): .libsubdirs
 
 libefi.a: $(OBJS)
 	$(AR) $(ARFLAGS) $@ $^
